lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -127,10 +127,9 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     const metricAuditsEl = this.renderAuditGroup(groups.metrics);
 
     // Metric descriptions toggle.
-    const auditGroupHeader = this.dom.find('.lh-audit-group__header', metricAuditsEl);
     const toggleTmpl = this.dom.cloneTemplate('#tmpl-lh-metrics-toggle', this.templateContext);
     const toggleEl = this.dom.find('.lh-metrics-toggle', toggleTmpl);
-    auditGroupHeader.appendChild(toggleEl);
+    metricAuditsEl.prepend(...toggleEl.childNodes);
```

Comment:
did we lose a little padding by removing this nesting layer? feels a bit close now

![image](https://user-images.githubusercontent.com/2301202/57173803-49957680-6e03-11e9-82a2-6949601c02e2.png)

Reply:
thx.